hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkGetByKeyHandler.java

```diff
@@ -57,13 +60,13 @@ public Response createEmptyResponse(@NonNull final ResponseHeader header) {
     @Override
     public void validate(@NonNull final QueryContext context) throws PreCheckException {
         requireNonNull(context);
-        throw new UnsupportedOperationException("Not implemented");
+        throw new UnsupportedOperationException("Not supporting");
```

Comment:
If unsupported, then this should really throw a PreCheckException with an UNSUPPORTED code.

Reply:
There's a separate PR for that (#6949 )